### DISCUSSION TITLE
Real-ESRGAN

### DISCUSSION THREAD

**Discussion by besalarana** [xinntao/Real-ESRGAN#503] [Q&A]
hi i
 Real-ESRGAN  s AMAZING! thank YOU, but in this link
 https://huggingface.co/spaces/akhaliq/Real-ESRGAN
  page is NOT workinh¡g, do you l¡knw why?
 please help me!

**Comment by lightshadown** [comment]
that page is not using the actual repo, its from another repo and meabe the training data on that one is totally diferent, i did try it and got horrible results, try using the colab from this repo on the frontpage, it might work for you
